ui/tests/e2e/test.js

```diff
@@ -168,6 +168,27 @@ describe("End-to-end", () => {
     await waitForFacetsUpdate(3500);
   });
 
+  test("Search box - values", async () => {
```

Comment:
Delete `test("Search box - click on drop-down row")`. This test already clicks on drop-down row.